packages/core/theme-module/theme/pages/MyAccount/OrderHistory.vue

```diff
@@ -30,7 +30,9 @@
           </SfTableData>
           <SfTableData class="orders__view">
             <SfButton class="sf-button--text color-secondary mobile-only">Download</SfButton>
-            <SfButton class="sf-button--text color-secondary desktop-only">VIEW</SfButton>
+            <nuxt-link :to="{ path: `/my-account/order-details/${orderGetters.getId(order)}`}">
```

Comment:
use localized link ;)

Reply:
Can you explain? I haven't seen any localized url in the project